### PR TITLE
Start-of-cycle 46 basic dependency version bump

### PR DESCRIPTION
**Pull request by john-metabase**
This is a basic dependency bump. Some dependencies still have available major version updates, but since those would likely introduce breaking changes, they have been updated to only the most recent version available of the same major version (e.g. Jetty).

Some minor changes to MongoDB and SQLite tests were required.

### CODE REVIEW

**Comment by john-metabase** [comment]
PS most changes were managed with: `clojure -Sdeps '{:deps {com.github.liquidz/antq {:mvn/version "RELEASE"}}}' -M -m antq.core --upgrade`

Some updates to new major/questionable versions were skipped and I tracked down the latest legitimate/working releases.

**Comment by john-metabase** [comment]
antq also supports updating our GitHub actions. I have not included those changes in this PR yet. Do we want to include those in the bump? That may be more appropriate for a separate PR.

File: deps.edn [hunk 1]
```diff
@@ -195,22 +198,21 @@
    {clj-http-fake/clj-http-fake  {:mvn/version "1.0.3"
                                   :exclusions  [slingshot/slingshot]}
     clj-kondo/clj-kondo          {:mvn/version "2022.08.03"}                ; this is not for RUNNING kondo, but so we can hack on custom hooks code from the REPL.
-    cloverage/cloverage          {:mvn/version "1.2.2"}
+    cloverage/cloverage          {:mvn/version "1.2.4"}
     com.gfredericks/test.chuck   {:mvn/version "0.2.13"}                    ; generating strings from regexes (useful with malli)
-    djblue/portal                {:mvn/version "0.30.0"}                    ; ui for inspecting values
```
Comment (camsaul) [inline deps.edn:200]: did you mean to remove the comments here?

File: deps.edn [hunk 2]
```diff
@@ -195,22 +198,21 @@
...
-    eftest/eftest                {:mvn/version "0.5.9"}
+    djblue/portal                {:mvn/version "0.35.0"}
+    eftest/eftest                {:mvn/version "0.6.0"}
     io.github.camsaul/humane-are {:mvn/version "1.0.2"}
-    jonase/eastwood              {:mvn/version "1.2.2"}                     ; inspects namespaces and reports possible problems using tools.analyzer
-    lambdaisland/deep-diff2      {:mvn/version "2.3.127"}                   ; way better diffs
-    methodical/methodical        {:mvn/version "0.15.0.1"}                  ; drop-in replacements for Clojure multimethods and adds several advanced features
```
Comment (camsaul) [inline deps.edn:205]: more deleted comments

**Review by camsaul — APPROVED** [review]
looks good to me. Can you open GitHub issues for the major version bumps or anything you had to revert here (if you haven't done so already?)

File: deps.edn (hunk 1 above)
Comment (john-metabase) [reply deps.edn:200]: Oops, no, that was `antq`. Will restore them.

**Comment by john-metabase** [comment]
@camsaul I've created a few and will be creating the rest shortly

**Pull request merged**